api/internal/handler/plugin/plugin_test.go

```diff
@@ -55,4 +55,48 @@ func TestPlugin(t *testing.T) {
 	ctx.SetInput(input2)
 	val, _ = handler.Schema(ctx)
 	assert.Nil(t, val)
+
+	/*
+	 get plugin schema with schema_type: consumer
+	 plugin has consumer_schema
+	 return plugin`s consumer_schema
+	*/
+	input3 := &GetInput{}
```

Comment:
can we reuse one variable, eg: `input` ?

Reply:
done~!